elements/bulbs-slideshow/bulbs-slideshow.js

```diff
@@ -103,9 +109,18 @@ class BulbsSlideshow extends BulbsHTMLElement {
 
   afterSlideTransition () {
     this.render();
+    this.sendPageView.bind(this).call();
     this.navigateToNextSlide();
   }
 
+  sendPageView () {
+    this.analyticsManager.trackPageView(
```

Comment:
This `sendPageView` method is duplicated in enough places where it makes sense to wonder if we should bubble this up to a util as well, and pass in the relevant values?